Sources/Xcodeproj/pbxproj().swift

```diff
@@ -15,9 +15,10 @@
 import PackageType
 import Utility
 
-public func pbxproj(srcroot srcroot: String, projectRoot: String, modules: [SwiftModule], externalModules: [SwiftModule], products _: [Product], printer print: (String) -> Void) {
+public func pbxproj(srcroot srcroot: String, projectRoot: String, modules: [XcodeModuleProtocol], externalModules: [XcodeModuleProtocol], products _: [Product], printer print: (String) -> Void) {
 
-    let rootModulesSet = Set(modules).subtract(Set(externalModules))
+    // let rootModulesSet = Set(modules).subtract(Set(externalModules))
```

Comment:
The only problem is this 2 lines. As I understood, I can't do this for protocols?